otx/algorithms/detection/tasks/inference.py

```diff
@@ -226,28 +229,33 @@ def _init_recipe(self):
         if train_type not in (TrainType.SEMISUPERVISED, TrainType.INCREMENTAL):
             raise NotImplementedError(f"Train type {train_type} is not implemented yet.")
         if train_type == TrainType.SEMISUPERVISED:
-            if self._data_cfg.get("data", None) and self._data_cfg.data.get("unlabeled", None):
-                recipe = os.path.join(recipe_root, "semisl.py")
+            if self._is_training:
+                if self._data_cfg.get("data", None) and self._data_cfg.data.get("unlabeled", None):
+                    recipe = os.path.join(recipe_root, "semisl.py")
+                    self.base_dir = os.path.join(self.template_dir, "semisl")
```

Comment:
This seems to be a problem with templates generated by `otx build`. Even if the folder created by otx build is semi, it will consist of the same file structure as supervised.

Reply:
This will be resolved using builder class

Reply:
We put this as a temporary measure, and agreed to have a subdirectory structure even after the build. If anyone sees this and has ideas, please leave a comment.